File: app/src/main/java/com/aait/aec/data/network/model/exam/Student.java

package com.aait.aec.data.network.model.exam;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Student {

    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("ID")
    @Expose
    private String iD;
    @SerializedName("result")
    @Expose
    private String result;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getID() {
        return iD;
    }

    public void setID(String iD) {
        this.iD = iD;
    }

    public String getResult() {
        return result;
    }

    public void setResult(String result) {
        this.result = result;
    }

}

File: app/src/main/java/com/aait/aec/data/db/model/Student.kt
package com.aait.aec.data.db.model

class Student {
    var row: Int? = null
    var name: String? = null
    var id: String? = null
    var score: Double? = null

    constructor(row: Int?, name: String?, id: String?, score: Double) {
        this.row = row
        this.name = name
        this.id = id
        this.score = score
    }
}